client/src/rpc_client.rs

```diff
@@ -1264,7 +1264,10 @@ impl RpcClient {
         self.send_and_confirm_transaction_with_spinner_and_config(
             transaction,
             commitment,
-            RpcSendTransactionConfig::default(),
+            RpcSendTransactionConfig {
+                preflight_commitment: Some(commitment.commitment),
```

Comment:
im not familar with the rust client. is it intentional to not do something like `options.preflightCommitment || options.commitment` here?

Reply:
In this case the user can no way to provide `preflightCommitment` into the function.